### PR TITLE
Add brokers scraper boiler plate

### PR DESCRIPTION
**Pull request by dshomoye**
**Description:** This is an incremental PR that adds the boiler plate code for the brokers portion of the kafkametrics receiver. No actual metrics collection is implemented.

**Testing:** - unit tests added for added functions

### CODE REVIEW

File: receiver/kafkametricsreceiver/scraper_test_helper.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	s.Called()
+	return s.closed()
+}
+
+func (s *mockSaramaClient) Close() error {
+	s.Called()
+	return s.close()
+}
+
+func getMockClient() *mockSaramaClient {
```
Comment (jrcamp) [inline receiver/kafkametricsreceiver/scraper_test_helper.go:50]: newMockClient maybe?

File: receiver/kafkametricsreceiver/scraper_test_helper.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+func mockNewSaramaClient([]string, *sarama.Config) (sarama.Client, error) {
+	return getMockClient(), nil
+}
+
+type mockSaramaClient struct {
+	mock.Mock
+	sarama.Client
+
+	close  func() error
+	closed func() bool
```
Comment (jrcamp) [inline receiver/kafkametricsreceiver/scraper_test_helper.go:37]: Do they need to be functions or just values to be returned, e.g. `close error`

**Review by jrcamp — COMMENTED** [review]
Just a couple nits. btw when you go to add metrics be sure to use https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/metric-metadata.md

File: receiver/kafkametricsreceiver/scraper_test_helper.go (hunk 2 above)
Comment (dshomoye) [reply receiver/kafkametricsreceiver/scraper_test_helper.go:37]: you're absolutely right! they don't, I had to test it out to know. Something new...

**Comment by dshomoye** [comment]
> Just a couple nits. btw when you go to add metrics be sure to use https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/metric-metadata.md

I hadn't seen this before I will be sure to add it as well.

**Review by jrcamp — APPROVED** [review]

**Comment by bogdandrutu** [comment]
Please rebase

**Comment by dshomoye** [comment]
> Please rebase

Done

**Pull request merged**